pkg/http/authorization_test.go

```diff
@@ -1,187 +1,222 @@
 package http
```

Comment:
Unit tests will also be simplified, because they'll have to exercise only the `validateJWTAndGetScopes` function.